src/loaders/LoaderUtils.js

```diff
@@ -1,39 +1,5 @@
 class LoaderUtils {
 
-	static decodeText( array ) {
```

Comment:
I did not meant to completely remove `decodeText()`. Just the fallback when `TextDecoder` isn't available.

Reply:
Since #25236, there's no codes depending on this API in whole threejs codebase

Reply:
Then we should deprecate the method first before it can be removed ten releases later.

Reply:
good point. :D